config/settings.development.json

```diff
@@ -56,16 +56,12 @@
   ],
 
   "defaultSmartContract": [
-    { "name": "Kapiolani Blvd Unit", "homeownerName": "Naruto Uzumaki", "homeownerEmail": "naruto@foo.com", "homeownerPhoneNumber": "808-106-9999", "homeownerSignature": "", "tenantName": "John Doe", "tenantEmail": "john@foo.com", "tenantPhoneNumber": "808-234-5678", "tenantStance": "Unsigned", "tenantSignature": "", "unitAddress": "123 Kapiolani Blvd Honolulu, HI 96826", "monthlyRent": 1000, "status": "Pending", "termsAndConditions": "Terms and conditions for smart contract with 1 tenet. Cost, frequency of payment, parties involved.", "owner": "naruto@foo.com"
+    { "name": "Kapiolani Blvd Unit", "homeownerName": "Naruto Uzumaki", "homeownerEmail": "naruto@foo.com", "homeownerPhoneNumber": "808-106-9999", "homeownerSignature": "", "tenantName": "John Doe", "tenantEmail": "john@foo.com", "tenantPhoneNumber": "808-234-5678", "tenantStance": "Unsigned", "tenantSignature": "", "unitAddress": "123 Kapiolani Blvd Honolulu, HI 96826", "monthlyRent": 1000, "status": "Active", "termsAndConditions": "Terms and conditions for smart contract with 1 tenet. Cost, frequency of payment, parties involved.", "owner": "naruto@foo.com"
```

Comment:
Why do we need name? Can't we just use unitAddress?

Reply:
I think the name is the name of the Contract?  where the unitAddress is just the address